.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+name: Actions
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Dependencies
+      run: npm install
+    - name: Run linters
+      run: npm run lint
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    needs: lint
+    strategy:
+      matrix:
+        node-version: [15.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm test
+      env:
+        CI: true
+  release:
```

Comment:
It appears that this job is running all the time (though it should only actually trigger the release when actually on the `master` branch).  But still, this is not ideal.  Seems like we need to find some way to skip this job except when it is actually needed.  

I think we should be able to use the `if:` field to specify some logic about when this job should run: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
release's release actions are controlled inside of semantic-release and follow our config in the package.json